### DISCUSSION TITLE
Bromite/Ungoogled-Chromium

### DISCUSSION THREAD

**Discussion by musa555ev** [bromite/bromite#1639] [Q&A]
https://user-images.githubusercontent.com/95716820/145097399-8a0355d8-96fa-4d74-a57d-470df298e86e.mp4

The site freezes in the browser Bromite, this does not happen in ungoogled chromium. Why?

**Comment by uazo** [comment]
could you try to disable the android autofill flag under password settings and tell us if anything changes?

**Reply by musa555ev** [reply]
in the video it is already turned off

**Comment by csagan5** [comment]
Most probably this website is trying to fingerprint you, or some other functionality blocked via ad filters. Follow the steps at https://developer.chrome.com/docs/devtools/remote-debugging/ to see what Javascript errors are there.
